lib/krane/cli/deploy_command.rb

```diff
@@ -60,10 +60,10 @@ def self.from_options(namespace, context, options)
             namespace: namespace,
             context: context,
             current_sha: options['current-sha'],
-            template_paths: paths,
+            filenames: paths,
             bindings: bindings_parser.parse,
             logger: logger,
-            max_watch_seconds: ::Krane::DurationParser.new(options["global-timeout"]).parse!.to_i,
+            global_timeout: ::Krane::DurationParser.new(options["global-timeout"]).parse!.to_i,
             selector: selector,
             protected_namespaces: protected_namespaces,
           )
```

Comment:
I can't comment on line 73. But should `allow_protected_ns: !protected_namespaces.empty?,` be renamed too?

Reply:
This actually could be removed from the public interface, right? This is maintained as an argument mostly for backwards compatibility with `kubernetes-deploy`.

Reply:
`allow_protected_ns` can be removed.

Reply:
OK, will remove on https://github.com/Shopify/kubernetes-deploy/pull/622.